src/styles/base.css

```diff
@@ -150,6 +150,10 @@
     position: relative !important; /* Override the absolute positioning of the play button (TODO: remove this hack) */
 }
 
+pre.ap-terminal {
+  transform: translateZ(0);
+}
```

Comment:
Moving this into the base layer and using tailwinds @apply directive would be awesome to keep everything consistent

Reply:
I'm not super familiar with tailwind but I'm not sure if this is what you meant https://github.com/NixOS/nixos-homepage/pull/1378/commits/a1ccacd5092361be4497272d6cf4e7d5e0f9744d. I had to add the asciinema-player js to tailwind's content scan otherwise the class would get stripped out